### ISSUE TITLE
Convert site titles to use the new block format (H1 on home page, P elsewhere)

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#62]
Len Burns feedback. This is a real issue, and we need to dive deeper. Rob has provided a Class to allow people to get the look they want without using an H1. 
Create a bug, but assess the solution only - if this is too much work, we may defer.

**Comment by dajoturner** [comment]
First part of this was done in #57. What's left is to implement the block format that creates an H1 on the site home page, but uses <P> tags everywhere else. Rob made progress on this, but ran into a roadblock. Current state is that every site has one minor accessibility violation on the home page. Determined to be a medium priority.
